cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -887,6 +887,10 @@ public static String getZKQuorum(CConfiguration cConf) {
       //For task worker
       public static final String CLASS = "clz";
       public static final String TRIES = "try";
+
+      //For system app entity
+      public static final String SYSTEM_APP_ENTITY_TYPE = "sya";
```

Comment:
Let's move this around service, since these are related to service now. Also, let's call it `APP_ENTITY_TYPE` and `APP_ENTITY_TYPE_NAME` so in case in future, user defined service can also use these

Reply:
ack